recipes/datarobot-bp-workshop/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "datarobot-bp-workshop" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datarobot-bp-workshop-{{ version }}.tar.gz
+  sha256: 0f5e94206d89ea73d5798c88e145e3b2d8ebfadb258f30ca6374259813257582
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest
```

Comment:
Is `pytest` required to build the package? I see that it's imported within a method, but I don't think it's ever actually run.